### PR TITLE
Review fixes

### PR DESCRIPTION
**Pull request by poljar**
This PR fixes most of the issues raised in our initial review.
Most notably #7 and #9 are not addressed yet.

### CODE REVIEW

**Comment by poljar** [comment]
#9 has been addressed now as well.

File: src/index.rs [hunk 1]
```diff
@@ -19,6 +19,18 @@ use tantivy::tokenizer::Tokenizer;
 use crate::japanese_tokenizer::TinySegmenterTokenizer;
 use crate::types::{Event, EventId, EventType, Language, SearchConfig};
 
+// Tantivy requires at least of 3MB per writer thread, Tantivy will panic if we
```
Comment (jryans) [inline src/index.rs]:
```suggestion
// Tantivy requires at least 3MB per writer thread and will panic if we
```

**Comment by poljar** [comment]
God damn, left a typo in the commit message... This branch doesn't have the CI fixes.

**Comment by jryans** [comment]
Could always pull the PR branch locally and force push with a fixed message if you really want... 😜

**Comment by poljar** [comment]
This has been merged after the typo from the last commit has been fixed.
Merge commit is https://github.com/matrix-org/seshat/commit/68d76b9678486a4a761957058795ba466027f1ea

**Pull request closed without merging**